packages/sanity/src/core/form/inputs/BooleanInput.tsx

```diff
@@ -30,32 +29,35 @@ export function BooleanInput(props: BooleanInputProps) {
   const tone: CardTone | undefined = readOnly ? 'transparent' : undefined
 
   return (
-    <Card border data-testid="boolean-input" radius={2} tone={tone}>
-      <Flex>
-        <ZeroLineHeightBox padding={3}>
-          <LayoutSpecificInput
-            label={schemaType.title}
-            {...elementProps}
-            checked={checked}
-            disabled={readOnly}
-            indeterminate={indeterminate}
-            style={{margin: -4}}
-          />
-        </ZeroLineHeightBox>
-        <Box flex={1} paddingY={3}>
-          <FormFieldHeaderText
-            description={schemaType.description}
-            inputId={id}
-            validation={validation}
-            title={schemaType.title}
-          />
-        </Box>
-        <CenterAlignedBox paddingX={3} paddingY={1}>
-          <FormFieldStatus maxAvatars={1} position="top">
-            {/*<FieldPresence maxAvatars={1} presence={presence} />*/}
-          </FormFieldStatus>
-        </CenterAlignedBox>
-      </Flex>
-    </Card>
+    <>
```

Comment:
Probably an oversight, do we need this empty fragment? It's adding extra differences which seems not to be needed.

Reply:
ah good catch this is from a previous iteration attempt that got leftover, thanks!